src/Views/LandingPage.js

```diff
@@ -1,3 +1,47 @@
+<<<<<<< HEAD
```

Comment:
Here is a merge conflict that needs to be resolved. You can find more information about resolving conflicts [here](https://profy.dev/project/reddit-timer/merge).

I'd suggest you create a new branch `fix-header` from main (after running `git pull` there). In that branch you can resolve this conflict and the other one.

Let me know if you need more help